bindings/core/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Core library for iota-sdk bindings
+
+mod error;
+mod method;
+pub mod method_handler;
```

Comment:
Was it public for the Result type? Should not be needed

Reply:
Also for mqtt listen, but exposed that also in a different place now